K2Bridge/Factories/AggregateFactory.cs

```diff
@@ -183,10 +208,12 @@ public static void AddAggregates(this AggregateDictionary aggregateDictionary, s
                         var key = columnMetadata[0];
                         aggregateDictionary.Add(key, GetExtendedStatsAggregate(column.ColumnName, columnMetadata, row, logger));
                     }
+                    /*
```

Comment:
why is this commented out?

Reply:
The code bloc is deleted as it's throwing exception when it's histogram

Reply:
Then add another branch to the if, don't leave commented out code inside.

Reply:
You now just deleted the code, that's still not good - add a branch for histogram, leave in the default case.

Reply:
Done